frontend/src/pages/WhatIsHealthEquity/WhatIsHealthEquityPage.module.scss

```diff
@@ -383,10 +355,21 @@
     font-size: 35px;
   }
 
+  .CopyIcon {
+    color: $alt-green;
+    height: 35px;
+  }
+
   .ContactUsLink {
     color: $alt-green;
     font-weight: 500;
     font-size: 26px;
     line-height: 31.47px;
   }
 }
+
+.TextField {
+  .MuiOutlinedInput-root {
+    border-radius: 40px !important;
```

Comment:
It doesn't look like this styling applies to the share text field in my netlify preview, could you try moving it back to EmailAddressFormText / EmailTextField?

Reply:
oh oops I meant to delete this good catch